src/app/options/options.component.spec.ts

```diff
@@ -88,13 +93,16 @@ describe('OptionsComponent', () => {
     expect((await options[0].getText())).toEqual("test1");
   });
 
-  it('should update additional options string from text in additional options textbox', async () => {
+  it('should update data$ observable from additional options string', async () => {
     await fixture.whenStable();
     const additionalOptionsHarness = await loader.getHarness(MatInputHarness.with(
       {selector: '.additionalOptions'}));
+    let data: OpenBabelData = blankData;
+    component.data$ = of(blankData).pipe(tap((thisData) => data = thisData));
```

Comment:
same question. if component's data comes from the service, it should already have this default, so why does it need to be set here?